python310Packages.fsspec: 2022.10.0 -> 2023.1.0, python310Packages.gcsfs: 2022.10.0 -> 2023.1.0

fabaff opened the pull request:
###### Description of changes

- https://github.com/fsspec/filesystem_spec/raw/2023.1.0/docs/source/changelog.rst
- https://github.com/fsspec/gcsfs/raw/2023.1.0/docs/source/changelog.rst

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python310Packages.fsspec: 2022.10.0 -> 2023.1.0

Changelog: https://github.com/fsspec/filesystem_spec/raw/2023.1.0/docs/source/changelog.rst

fabaff committed:
python310Packages.gcsfs: 2022.10.0 -> 2023.1.0

Changelog: https://github.com/fsspec/gcsfs/raw/2023.1.0/docs/source/changelog.rst

fabaff commented:
Fixes for failures:

- #221157

fabaff commented:
Result of `nixpkgs-review pr 221132` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>13 packages marked as broken and skipped:</summary>
  <ul>
    <li>intensity-normalization</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.pywick</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.rl-coach</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.shap</li>
    <li>python311Packages.pandas-stubs</li>
  </ul>
</details>
<details>
  <summary>33 packages failed to build:</summary>
  <ul>
    <li>databricks-sql-cli</li>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>openai-full</li>
    <li>parquet-tools</li>
    <li>python310Packages.apache-beam</li>
    <li>python310Packages.awswrangler</li>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.databricks-sql-connector</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.pandas-stubs</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pyvista</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.sfepy</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.vqgan-jax</li>
    <li>python310Packages.webdav4</li>
    <li>python311Packages.apache-beam</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.databricks-sql-connector</li>
    <li>python311Packages.ibis-framework</li>
    <li>python311Packages.webdav4</li>
  </ul>
</details>
<details>
  <summary>96 packages built:</summary>
  <ul>
    <li>easyocr (python310Packages.easyocr)</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>khoj</li>
    <li>larynx-train (python310Packages.larynx-train)</li>
    <li>napari (python310Packages.napari)</li>
    <li>ome_zarr (python310Packages.ome-zarr)</li>
    <li>openai-whisper (python310Packages.openai-whisper)</li>
    <li>python310Packages.adlfs</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.autofaiss</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.bsuite</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.dremel3dpy</li>
    <li>python310Packages.dvc-data</li>
    <li>python310Packages.dvc-http</li>
    <li>python310Packages.dvc-objects</li>
    <li>python310Packages.embedding-reader</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.evaluate</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.ocifs</li>
    <li>python310Packages.ossfs</li>
    <li>python310Packages.psd-tools</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyathena</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.recordlinkage</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.sentence-transformers</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.stanza</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.trainer</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.zcs</li>
    <li>python311Packages.adlfs</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.datafusion</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.db-dtypes</li>
    <li>python311Packages.dvc-data</li>
    <li>python311Packages.dvc-http</li>
    <li>python311Packages.dvc-objects</li>
    <li>python311Packages.embedding-reader</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.fsspec</li>
    <li>python311Packages.gcsfs</li>
    <li>python311Packages.google-cloud-bigquery</li>
    <li>python311Packages.ocifs</li>
    <li>python311Packages.ossfs</li>
    <li>python311Packages.pyarrow</li>
    <li>python311Packages.pyathena</li>
    <li>python311Packages.recordlinkage</li>
    <li>python311Packages.s3fs</li>
    <li>python311Packages.scmrepo</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.transformers</li>
    <li>streamlit</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tts</li>
  </ul>
</details>

mattmelling commented:
Moving in the right direction, most of the broken packages appear broken in master with the exception of webdav4.

Result of `nixpkgs-review pr 221132` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>15 packages marked as broken and skipped:</summary>
  <ul>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>intensity-normalization</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.pywick</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.rl-coach</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.shap</li>
    <li>python311Packages.pandas-stubs</li>
  </ul>
</details>
<details>
  <summary>30 packages failed to build:</summary>
  <ul>
    <li>databricks-sql-cli</li>
    <li>openai-full</li>
    <li>parquet-tools</li>
    <li>python310Packages.apache-beam</li>
    <li>python310Packages.awswrangler</li>
    <li>python310Packages.bsuite</li>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.databricks-sql-connector</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.pandas-stubs</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.vqgan-jax</li>
    <li>python310Packages.webdav4</li>
    <li>python311Packages.apache-beam</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.databricks-sql-connector</li>
    <li>python311Packages.ibis-framework</li>
    <li>python311Packages.webdav4</li>
  </ul>
</details>
<details>
  <summary>97 packages built:</summary>
  <ul>
    <li>easyocr (python310Packages.easyocr)</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>khoj</li>
    <li>larynx-train (python310Packages.larynx-train)</li>
    <li>napari (python310Packages.napari)</li>
    <li>ome_zarr (python310Packages.ome-zarr)</li>
    <li>openai-whisper (python310Packages.openai-whisper)</li>
    <li>python310Packages.adlfs</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.autofaiss</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.dremel3dpy</li>
    <li>python310Packages.dvc-data</li>
    <li>python310Packages.dvc-http</li>
    <li>python310Packages.dvc-objects</li>
    <li>python310Packages.embedding-reader</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.evaluate</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.ocifs</li>
    <li>python310Packages.ossfs</li>
    <li>python310Packages.psd-tools</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyathena</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.pyvista</li>
    <li>python310Packages.recordlinkage</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.sentence-transformers</li>
    <li>python310Packages.sfepy</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.stanza</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.trainer</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.zcs</li>
    <li>python311Packages.adlfs</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.datafusion</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.db-dtypes</li>
    <li>python311Packages.dvc-data</li>
    <li>python311Packages.dvc-http</li>
    <li>python311Packages.dvc-objects</li>
    <li>python311Packages.embedding-reader</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.fsspec</li>
    <li>python311Packages.gcsfs</li>
    <li>python311Packages.google-cloud-bigquery</li>
    <li>python311Packages.ocifs</li>
    <li>python311Packages.ossfs</li>
    <li>python311Packages.pyarrow</li>
    <li>python311Packages.pyathena</li>
    <li>python311Packages.recordlinkage</li>
    <li>python311Packages.s3fs</li>
    <li>python311Packages.scmrepo</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.transformers</li>
    <li>streamlit</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tts</li>
  </ul>
</details>

fabaff committed:
python310Packages.ossfs: 2023.1.0 -> 2023.3.0

Changelog: https://github.com/fsspec/ossfs/releases/tag/2023.3.0

fabaff commented:
Result of `nixpkgs-review pr 221132` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.mask-rcnn.dist</li>
    <li>python310Packages.rl-coach</li>
    <li>python310Packages.rl-coach.dist</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.rlax.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python310Packages.vqgan-jax</li>
    <li>python310Packages.vqgan-jax.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>171 packages failed to build:</summary>
  <ul>
    <li>databricks-sql-cli</li>
    <li>databricks-sql-cli.dist</li>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>dvc-with-remotes.dist</li>
    <li>dvc.dist</li>
    <li>easyocr (python310Packages.easyocr)</li>
    <li>easyocr.dist (python310Packages.easyocr.dist)</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>intensity-normalization (python310Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python310Packages.intensity-normalization.dist)</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>napari (python310Packages.napari)</li>
    <li>napari.dist (python310Packages.napari.dist)</li>
    <li>ome_zarr (python310Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python310Packages.ome-zarr.dist)</li>
    <li>parquet-tools</li>
    <li>parquet-tools.dist</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.aplpy.dist</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.batchgenerators.dist</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.boxx.dist</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.bpycv.dist</li>
    <li>python310Packages.bsuite</li>
    <li>python310Packages.bsuite.dist</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffe.bin</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.caffeWithCuda.bin</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-awkward.dist</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-image.dist</li>
    <li>python310Packages.databricks-sql-connector</li>
    <li>python310Packages.databricks-sql-connector.dist</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.datashader.dist</li>
    <li>python310Packages.dremel3dpy</li>
    <li>python310Packages.dremel3dpy.dist</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.env-canada.dist</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.glymur.dist</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.handout.dist</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.ibis-framework.dist</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.image-match.dist</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imagecorruptions.dist</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imageio.dist</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.imgaug.dist</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.moviepy.dist</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-console.dist</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.napari-svg.dist</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pims.dist</li>
    <li>python310Packages.psd-tools</li>
    <li>python310Packages.psd-tools.dist</li>
    <li>python310Packages.pyvista</li>
    <li>python310Packages.pyvista.dist</li>
    <li>python310Packages.pywick</li>
    <li>python310Packages.pywick.dist</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.qiskit-machine-learning.dist</li>
    <li>python310Packages.qiskit.dist</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.runway-python.dist</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scikitimage.dist</li>
    <li>python310Packages.sfepy</li>
    <li>python310Packages.sfepy.dist</li>
    <li>python310Packages.shap</li>
    <li>python310Packages.shap.dist</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.slicedimage.dist</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.spectral-cube.dist</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.streamz.dist</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.stytra.dist</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.sunpy.dist</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.test-tube.dist</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tifffile.dist</li>
    <li>python310Packages.webdav4</li>
    <li>python310Packages.webdav4.dist</li>
    <li>python310Packages.zcs</li>
    <li>python310Packages.zcs.dist</li>
    <li>python311Packages.apache-beam</li>
    <li>python311Packages.apache-beam.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.databricks-sql-connector</li>
    <li>python311Packages.databricks-sql-connector.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.ibis-framework</li>
    <li>python311Packages.ibis-framework.dist</li>
    <li>python311Packages.image-match</li>
    <li>python311Packages.image-match.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.imgaug</li>
    <li>python311Packages.imgaug.dist</li>
    <li>python311Packages.intensity-normalization</li>
    <li>python311Packages.intensity-normalization.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.ome-zarr</li>
    <li>python311Packages.ome-zarr.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.runway-python</li>
    <li>python311Packages.runway-python.dist</li>
    <li>python311Packages.scikitimage</li>
    <li>python311Packages.scikitimage.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
  </ul>
</details>
<details>
  <summary>188 packages built:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>larynx-train (python310Packages.larynx-train)</li>
    <li>larynx-train.dist (python310Packages.larynx-train.dist)</li>
    <li>openai-full</li>
    <li>openai-full.dist</li>
    <li>openai-whisper (python310Packages.openai-whisper)</li>
    <li>openai-whisper.dist (python310Packages.openai-whisper.dist)</li>
    <li>python310Packages.adlfs</li>
    <li>python310Packages.adlfs.dist</li>
    <li>python310Packages.apache-beam</li>
    <li>python310Packages.apache-beam.dist</li>
    <li>python310Packages.autofaiss</li>
    <li>python310Packages.autofaiss.dist</li>
    <li>python310Packages.awswrangler</li>
    <li>python310Packages.awswrangler.dist</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.casa-formats-io.dist</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.clifford.dist</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway.dist</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-glm.dist</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-jobqueue.dist</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-ml.dist</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-mpi.dist</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.dask-yarn.dist</li>
    <li>python310Packages.dask.dist</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datafusion.dist</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.datasets.dist</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.db-dtypes.dist</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.devito.dist</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.distributed.dist</li>
    <li>python310Packages.dvc-data</li>
    <li>python310Packages.dvc-data.dist</li>
    <li>python310Packages.dvc-http</li>
    <li>python310Packages.dvc-http.dist</li>
    <li>python310Packages.dvc-objects</li>
    <li>python310Packages.dvc-objects.dist</li>
    <li>python310Packages.embedding-reader</li>
    <li>python310Packages.embedding-reader.dist</li>
    <li>python310Packages.evaluate</li>
    <li>python310Packages.evaluate.dist</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fastparquet.dist</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.fsspec.dist</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.gcsfs.dist</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.google-cloud-bigquery.dist</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.intake-parquet.dist</li>
    <li>python310Packages.intake.dist</li>
    <li>python310Packages.mlflow</li>
    <li>python310Packages.mlflow.dist</li>
    <li>python310Packages.ocifs</li>
    <li>python310Packages.ocifs.dist</li>
    <li>python310Packages.ossfs</li>
    <li>python310Packages.ossfs.dist</li>
    <li>python310Packages.pandas-stubs</li>
    <li>python310Packages.pandas-stubs.dist</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyarrow.dist</li>
    <li>python310Packages.pyathena</li>
    <li>python310Packages.pyathena.dist</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyfftw.dist</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.pytorch-lightning.dist</li>
    <li>python310Packages.recordlinkage</li>
    <li>python310Packages.recordlinkage.dist</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.s3fs.dist</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.scmrepo.dist</li>
    <li>python310Packages.sentence-transformers</li>
    <li>python310Packages.sentence-transformers.dist</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.sparse.dist</li>
    <li>python310Packages.sshfs</li>
    <li>python310Packages.sshfs.dist</li>
    <li>python310Packages.stanza</li>
    <li>python310Packages.stanza.dist</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stumpy.dist</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.tokenizers.dist</li>
    <li>python310Packages.trainer</li>
    <li>python310Packages.trainer.dist</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.transformers.dist</li>
    <li>python311Packages.adlfs</li>
    <li>python311Packages.adlfs.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datafusion</li>
    <li>python311Packages.datafusion.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.db-dtypes</li>
    <li>python311Packages.db-dtypes.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dvc-data</li>
    <li>python311Packages.dvc-data.dist</li>
    <li>python311Packages.dvc-http</li>
    <li>python311Packages.dvc-http.dist</li>
    <li>python311Packages.dvc-objects</li>
    <li>python311Packages.dvc-objects.dist</li>
    <li>python311Packages.embedding-reader</li>
    <li>python311Packages.embedding-reader.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.fastparquet</li>
    <li>python311Packages.fastparquet.dist</li>
    <li>python311Packages.fsspec</li>
    <li>python311Packages.fsspec.dist</li>
    <li>python311Packages.gcsfs</li>
    <li>python311Packages.gcsfs.dist</li>
    <li>python311Packages.google-cloud-bigquery</li>
    <li>python311Packages.google-cloud-bigquery.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.mlflow</li>
    <li>python311Packages.mlflow.dist</li>
    <li>python311Packages.ocifs</li>
    <li>python311Packages.ocifs.dist</li>
    <li>python311Packages.ossfs</li>
    <li>python311Packages.ossfs.dist</li>
    <li>python311Packages.pyarrow</li>
    <li>python311Packages.pyarrow.dist</li>
    <li>python311Packages.pyathena</li>
    <li>python311Packages.pyathena.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.recordlinkage</li>
    <li>python311Packages.recordlinkage.dist</li>
    <li>python311Packages.s3fs</li>
    <li>python311Packages.s3fs.dist</li>
    <li>python311Packages.scmrepo</li>
    <li>python311Packages.scmrepo.dist</li>
    <li>python311Packages.sshfs</li>
    <li>python311Packages.sshfs.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>python311Packages.webdav4</li>
    <li>python311Packages.webdav4.dist</li>
    <li>streamlit</li>
    <li>streamlit.dist</li>
    <li>visidata</li>
    <li>visidata.dist</li>
  </ul>
</details>

fabaff committed:
python310Packages.fsspec: 2023.1.0 -> 2023.4.0

Diff: https://github.com/fsspec/filesystem_spec/compare/2023.1.0...2023.4.0

Changelog: https://github.com/fsspec/filesystem_spec/raw/2023.4.0/docs/source/changelog.rst

fabaff committed:
python310Packages.gcsfs: 2023.1.0 -> 2023.4.0

Diff: https://github.com/fsspec/gcsfs/compare/refs/tags/2023.1.0...2023.4.0

Changelog: https://github.com/fsspec/gcsfs/raw/2023.4.0/docs/source/changelog.rst

fabaff committed:
python310Packages.aiooss2: init at 0.2.5

fabaff committed:
python310Packages.tifffile: 20232.2.3 -> 2023.4.12

Changelog: https://github.com/cgohlke/tifffile/blob/v2023.4.12/CHANGES.rst

fabaff committed:
python310Packages.ossfs: 2023.1.0 -> 2023.4.0

Diff: fsspec/ossfs@refs/tags/2023.1.0...2023.4.0

Changelog: https://github.com/fsspec/ossfs/releases/tag/2023.4.0

fabaff committed:
python310Packages.oss2: 2.16.0 -> 2.17.0

Changelog: https://github.com/aliyun/aliyun-oss-python-sdk/releases/tag/2.17.0

fabaff commented:
Fixes of some failures:

- https://github.com/NixOS/nixpkgs/pull/229254


Most failures are related to missing support for SQLAlchemy > 2 and Python 3.11.

fabaff commented:
Result of `nixpkgs-review pr 221132` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>98 packages failed to build:</summary>
  <ul>
    <li>databricks-sql-cli</li>
    <li>databricks-sql-cli.dist</li>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>dvc-with-remotes.dist</li>
    <li>dvc.dist</li>
    <li>intensity-normalization (python310Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python310Packages.intensity-normalization.dist)</li>
    <li>openai-full</li>
    <li>openai-full.dist</li>
    <li>parquet-tools</li>
    <li>parquet-tools.dist</li>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-awkward.dist</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-image.dist</li>
    <li>python310Packages.databricks-sql-connector</li>
    <li>python310Packages.databricks-sql-connector.dist</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.datashader.dist</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.fastai</li>
    <li>python310Packages.fastai.dist</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.glymur.dist</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.imgaug.dist</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.mask-rcnn.dist</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pims.dist</li>
    <li>python310Packages.pywick</li>
    <li>python310Packages.pywick.dist</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.qiskit-machine-learning.dist</li>
    <li>python310Packages.qiskit.dist</li>
    <li>python310Packages.rl-coach</li>
    <li>python310Packages.rl-coach.dist</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.rlax.dist</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.runway-python.dist</li>
    <li>python310Packages.shap</li>
    <li>python310Packages.shap.dist</li>
    <li>python310Packages.spacy</li>
    <li>python310Packages.spacy-loggers</li>
    <li>python310Packages.spacy-loggers.dist</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.spacy-transformers.dist</li>
    <li>python310Packages.spacy.dist</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.spectral-cube.dist</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.stytra.dist</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.sunpy.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python310Packages.textacy</li>
    <li>python310Packages.textacy.dist</li>
    <li>python310Packages.textnets</li>
    <li>python310Packages.textnets.dist</li>
    <li>python310Packages.vqgan-jax</li>
    <li>python310Packages.vqgan-jax.dist</li>
    <li>python310Packages.wandb</li>
    <li>python310Packages.wandb.dist</li>
    <li>python310Packages.webdav4</li>
    <li>python310Packages.webdav4.dist</li>
    <li>python311Packages.apache-beam</li>
    <li>python311Packages.apache-beam.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.databricks-sql-connector</li>
    <li>python311Packages.databricks-sql-connector.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.imgaug</li>
    <li>python311Packages.imgaug.dist</li>
    <li>python311Packages.intensity-normalization</li>
    <li>python311Packages.intensity-normalization.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.runway-python</li>
    <li>python311Packages.runway-python.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python311Packages.webdav4</li>
    <li>python311Packages.webdav4.dist</li>
  </ul>
</details>
<details>
  <summary>303 packages built:</summary>
  <ul>
    <li>dbx</li>
    <li>dbx.dist</li>
    <li>easyocr (python310Packages.easyocr)</li>
    <li>easyocr.dist (python310Packages.easyocr.dist)</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>larynx-train (python310Packages.larynx-train)</li>
    <li>larynx-train.dist (python310Packages.larynx-train.dist)</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>napari (python310Packages.napari)</li>
    <li>napari.dist (python310Packages.napari.dist)</li>
    <li>ome_zarr (python310Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python310Packages.ome-zarr.dist)</li>
    <li>openai-whisper (python310Packages.openai-whisper)</li>
    <li>openai-whisper.dist (python310Packages.openai-whisper.dist)</li>
    <li>python310Packages.accelerate</li>
    <li>python310Packages.accelerate.dist</li>
    <li>python310Packages.adlfs</li>
    <li>python310Packages.adlfs.dist</li>
    <li>python310Packages.aiooss2</li>
    <li>python310Packages.aiooss2.dist</li>
    <li>python310Packages.apache-beam</li>
    <li>python310Packages.apache-beam.dist</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.aplpy.dist</li>
    <li>python310Packages.autofaiss</li>
    <li>python310Packages.autofaiss.dist</li>
    <li>python310Packages.awswrangler</li>
    <li>python310Packages.awswrangler.dist</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.batchgenerators.dist</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.boxx.dist</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.bpycv.dist</li>
    <li>python310Packages.bsuite</li>
    <li>python310Packages.bsuite.dist</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffe.bin</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.caffeWithCuda.bin</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.casa-formats-io.dist</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.clifford.dist</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway.dist</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-glm.dist</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-jobqueue.dist</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-ml.dist</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-mpi.dist</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.dask-yarn.dist</li>
    <li>python310Packages.dask.dist</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datafusion.dist</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.datasets.dist</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.db-dtypes.dist</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.devito.dist</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.distributed.dist</li>
    <li>python310Packages.dremel3dpy</li>
    <li>python310Packages.dremel3dpy.dist</li>
    <li>python310Packages.dvc-data</li>
    <li>python310Packages.dvc-data.dist</li>
    <li>python310Packages.dvc-http</li>
    <li>python310Packages.dvc-http.dist</li>
    <li>python310Packages.dvc-objects</li>
    <li>python310Packages.dvc-objects.dist</li>
    <li>python310Packages.embedding-reader</li>
    <li>python310Packages.embedding-reader.dist</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.env-canada.dist</li>
    <li>python310Packages.evaluate</li>
    <li>python310Packages.evaluate.dist</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fastparquet.dist</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.fsspec.dist</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.gcsfs.dist</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.google-cloud-bigquery.dist</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.handout.dist</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.ibis-framework.dist</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.image-match.dist</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imagecorruptions.dist</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imageio.dist</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.intake-parquet.dist</li>
    <li>python310Packages.intake.dist</li>
    <li>python310Packages.mlflow</li>
    <li>python310Packages.mlflow.dist</li>
    <li>python310Packages.mmcv</li>
    <li>python310Packages.mmcv.dist</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.moviepy.dist</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-console.dist</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.napari-svg.dist</li>
    <li>python310Packages.ocifs</li>
    <li>python310Packages.ocifs.dist</li>
    <li>python310Packages.oss2</li>
    <li>python310Packages.oss2.dist</li>
    <li>python310Packages.ossfs</li>
    <li>python310Packages.ossfs.dist</li>
    <li>python310Packages.pandas-stubs</li>
    <li>python310Packages.pandas-stubs.dist</li>
    <li>python310Packages.peft</li>
    <li>python310Packages.peft.dist</li>
    <li>python310Packages.psd-tools</li>
    <li>python310Packages.psd-tools.dist</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyarrow.dist</li>
    <li>python310Packages.pyathena</li>
    <li>python310Packages.pyathena.dist</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyfftw.dist</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.pytorch-lightning.dist</li>
    <li>python310Packages.pyvista</li>
    <li>python310Packages.pyvista.dist</li>
    <li>python310Packages.recordlinkage</li>
    <li>python310Packages.recordlinkage.dist</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.s3fs.dist</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scikitimage.dist</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.scmrepo.dist</li>
    <li>python310Packages.sentence-transformers</li>
    <li>python310Packages.sentence-transformers.dist</li>
    <li>python310Packages.sfepy</li>
    <li>python310Packages.sfepy.dist</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.slicedimage.dist</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.sparse.dist</li>
    <li>python310Packages.sshfs</li>
    <li>python310Packages.sshfs.dist</li>
    <li>python310Packages.stanza</li>
    <li>python310Packages.stanza.dist</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.streamz.dist</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stumpy.dist</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.test-tube.dist</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tifffile.dist</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.tokenizers.dist</li>
    <li>python310Packages.trainer</li>
    <li>python310Packages.trainer.dist</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.transformers.dist</li>
    <li>python310Packages.zcs</li>
    <li>python310Packages.zcs.dist</li>
    <li>python311Packages.adlfs</li>
    <li>python311Packages.adlfs.dist</li>
    <li>python311Packages.aiooss2</li>
    <li>python311Packages.aiooss2.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.awswrangler</li>
    <li>python311Packages.awswrangler.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datafusion</li>
    <li>python311Packages.datafusion.dist</li>
    <li>python311Packages.datasets</li>
    <li>python311Packages.datasets.dist</li>
    <li>python311Packages.db-dtypes</li>
    <li>python311Packages.db-dtypes.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.dvc-data</li>
    <li>python311Packages.dvc-data.dist</li>
    <li>python311Packages.dvc-http</li>
    <li>python311Packages.dvc-http.dist</li>
    <li>python311Packages.dvc-objects</li>
    <li>python311Packages.dvc-objects.dist</li>
    <li>python311Packages.embedding-reader</li>
    <li>python311Packages.embedding-reader.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.evaluate</li>
    <li>python311Packages.evaluate.dist</li>
    <li>python311Packages.fastparquet</li>
    <li>python311Packages.fastparquet.dist</li>
    <li>python311Packages.fsspec</li>
    <li>python311Packages.fsspec.dist</li>
    <li>python311Packages.gcsfs</li>
    <li>python311Packages.gcsfs.dist</li>
    <li>python311Packages.google-cloud-bigquery</li>
    <li>python311Packages.google-cloud-bigquery.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.ibis-framework</li>
    <li>python311Packages.ibis-framework.dist</li>
    <li>python311Packages.image-match</li>
    <li>python311Packages.image-match.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.mlflow</li>
    <li>python311Packages.mlflow.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.ocifs</li>
    <li>python311Packages.ocifs.dist</li>
    <li>python311Packages.ome-zarr</li>
    <li>python311Packages.ome-zarr.dist</li>
    <li>python311Packages.oss2</li>
    <li>python311Packages.oss2.dist</li>
    <li>python311Packages.ossfs</li>
    <li>python311Packages.ossfs.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pyarrow</li>
    <li>python311Packages.pyarrow.dist</li>
    <li>python311Packages.pyathena</li>
    <li>python311Packages.pyathena.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.recordlinkage</li>
    <li>python311Packages.recordlinkage.dist</li>
    <li>python311Packages.s3fs</li>
    <li>python311Packages.s3fs.dist</li>
    <li>python311Packages.scikitimage</li>
    <li>python311Packages.scikitimage.dist</li>
    <li>python311Packages.scmrepo</li>
    <li>python311Packages.scmrepo.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.sshfs</li>
    <li>python311Packages.sshfs.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.tokenizers</li>
    <li>python311Packages.tokenizers.dist</li>
    <li>python311Packages.transformers</li>
    <li>python311Packages.transformers.dist</li>
    <li>streamlit</li>
    <li>streamlit.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>visidata</li>
    <li>visidata.dist</li>
  </ul>
</details>

bcdarwin commented:
Looks reasonable to me -- I didn't check them all but most of these other packages are already broken.  I don't know whether this PR should be considered for merging to `staging*` rather than `master`, though.

Luflosi commented:
Is there a specific reason why the imageio update isn't its own PR?

fabaff commented:
> Is there a specific reason why the imageio update isn't its own PR?

Because of https://github.com/imageio/imageio/blob/master/setup.py#L191. I can move it out to get faster `nix-review` results.

fabaff commented:
I would like to keep it targetting `master` as in a couple of days starts ZHF and it would make it simpler to get `dvc` fixed.

bcdarwin reviewed (approved):
Changes LGTM.
